Automattic/newspack-popups
# feat: retrieve existing WC orders data as donations

### All Submissions:






### Changes proposed in this Pull Request:



Closes #313.

### How to test the changes in this Pull Request:

1. Create a subscriber user, and a campaign with a segment set to donors
2. Visit the site as the logged-in subscriber user, observe the campaign is not displayed
3. Add an order (a donation) via WP Admin, in the name of the subscriber user
4. Visit the site, after subsequent visit observe the for-donors campaign is visible for the subscriber user

### Other information:
